### DISCUSSION TITLE
Meeting 10/18/2024

### DISCUSSION THREAD

**Discussion by Woodyaka** [Woodyaka/Mphys-Project#3] [General]
An overview of the meeting on 10/18/2024.
Discussed what a lit review is and what it should present.
Talked about running Tasha's code, and to ensure we can reproduce the figures from her paper.
Talked more specifically about what we have read.

**Comment by Woodyaka** [comment]
@dan800 

I have just cloned Tasha's code to my machine at home. I am not able to recreate any of the figures, due to the fact they all use local files, e.g. file = f'Nc_Files/SpE_Output/Wuhu_IonTr_run_SpE_Output_Jun-Aug_{WACCM_data_alt}_{filename_append}.nc'. I cant find these files any where. I am currently waiting for my Linux machine to update so I can get these running on ARC4, but I will need to know where these files are.

**Reply by dan800** [reply]
Data should be accessible here: /uolstore/Research/b/b0243/Data/Sporadic_E/
